securedrop_client/queue.py

```diff
@@ -19,67 +20,80 @@
 
 class RunnableQueue(QObject):
 
+    '''
+    Signal that is emitted BEFORE a queue is paused
+    '''
+    pause = pyqtSignal()
+
     def __init__(self, api_client: API, session_maker: scoped_session) -> None:
+        '''
```

Comment:
can we leave this comment near `self.order_number = itertools.count()` (since it's there to explain why we have this counter construction in our queues)

Reply:
yes, i'll move it back. in general, i think it's easier to read code that has more class-level and function-level comments than in-line comments, but this comment does seems to come out of nowhere in its new location.